src/main/java/catan/domain/model/dashboard/HexBean.java

```diff
@@ -131,6 +132,17 @@ public void setNodes(VerticalLinks<NodeBean> nodes) {
         this.nodes = nodes;
     }
 
+    public Set<NodeBean> getNodesWithBuildings() {
```

Comment:
it is not a good practice to name methods like field accessor (starting from 'get' + 'NameOfField'), in beans that are mapped to database, as different libraries for db-mapping can think that 'getNodesWithBuildings' is an accessor for field with name 'nodesWithBuildings', and will throw an exception when such field is not found